src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -53,4 +95,34 @@ private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Pre
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
 
+    /**
+     * Throws ParseException when one of the required prefixes for {@code Patient} are absent.
+     */
+    private static void arePrefixesPresentPatient(ArgumentMultimap argMultimap) throws ParseException {
+        if (!arePrefixesPresent(argMultimap, PREFIX_TYPE, PREFIX_NRIC, PREFIX_AGE, PREFIX_NAME, PREFIX_PHONE)
```

Comment:
no need `PREFIX_TYPE` cos we know the type is `patient ` alr

Reply:
Rectified in commit "Improve coding style of AddCommandParser.java"